docs/animation_during_reroute.md

```diff
@@ -0,0 +1,24 @@
+# Animation during reroute
+
```

Comment:
- add example animation
- more info about logic of the spinner appearance

Reply:
- Rm clone of the DOM nodes in we have no spinner
- Mention importance of the overlay